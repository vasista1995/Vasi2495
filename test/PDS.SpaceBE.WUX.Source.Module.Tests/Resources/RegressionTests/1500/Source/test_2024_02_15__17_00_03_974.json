{
  "Equipment": "SRMS04",
  "ParameterClass": "E_SPC",
  "Segment": "TDSON8",
  "Module": "M2_TEST",
  "Package": "PG-TDSON-8-66",
  "FourDReport": "*",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 200697763,
      "Seqnr": 2,
      "Value": 0.849,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 200697763,
      "Seqnr": 1,
      "Value": 0.816,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    }
  ],
  "SalesName": "BSC093N15NS5",
  "ChipType": "P9222C21311",
  "Track": "T1",
  "Operator": "30040342",
  "Baunumber": "90043423",
  "Line": "TDSON8",
  "Shift": "C",
  "Process": "Testing",
  "UniqueSourceKey": "WUX:BE:ZF323051C06:TCWUS:P_Test_Scan_Pack:Peel_Of_Force:50614:200697763:0",
  "PKey": "WUX:BE:ZF323051C06:TCWUS:P_Test_Scan_Pack:Peel_Of_Force:50614:200697763:0",
  "ParameterName": "Peel_Of_Force",
  "ParameterUnit": "N",
  "SampleDate": "2023-10-06T15:16:52Z",
  "SampleDateUtc": "2023-10-06T07:16:52Z",
  "SampleSize": 2,
  "Sigma": ",0233345237791561",
  "Mean": ",8325",
  "Median": ",8325",
  "Min": ",816",
  "Max": ",849",
  "Q1": ",82425",
  "Q3": ",84075",
  "LdsID": "1500",
  "IsCalcParam": 0,
  "MeanCntrlLow": 0.576785692454335,
  "MeanCntrlTarget": 0.772760751059963,
  "MeanCntrlHigh": 0.968735809665591,
  "CntrlLow": 0.545228027822357,
  "CntrlTarget": 0.772760751059963,
  "CntrlHigh": 1.00029347429757,
  "RangeCntrlTarget": 0.0480181708055726,
  "RangeCntrlHigh": 0.2796513136687,
  "SigmaCntrlTarget": 0.0363419972369961,
  "SigmaCntrlHigh": 0.168815409461328,
  "SpecLow": 0.4,
  "SpecTarget": 0.75,
  "SpecHigh": 1.1,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "Y",
  "CtrlHighEnabled": "Y",
  "CtrlRangeLowEnabled": "N",
  "CtrlRangeHighEnabled": "Y",
  "CtrlSigmaLowEnabled": "N",
  "CtrlSigmaCenterEnabled": "Y",
  "CtrlRangeCenterEnabled": "Y",
  "CtrlCenterEnabled": "Y",
  "CtrlSigmaHighEnabled": "Y",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "Y",
  "CreatedDate": "2023-10-06T15:16:52Z",
  "CreatedDateUtc": "2023-10-06T07:16:52Z",
  "UpdatedDate": "2023-10-06T15:16:52Z",
  "UpdatedDateUtc": "2023-10-06T07:16:52Z",
  "RvStoreFlag": "Y",
  "Operation": "P_Test_Scan_Pack",
  "Facility": "TCWUS",
  "Lot": "ZF323051C06",
  "ChannelId": "50614",
  "CkcId": "0",
  "ChannelName": "Tape_PeelOffForce/TDSON8",
  "ChannelDescr": "Created from LDS \nexclude list by \naction 'Create SPC Channel'",
  "SpaceInstanceName": "BE",
  "Sitekey": "WUX",
  "CalcCreated": "11/22/2022 16:25:53",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": " ",
  "SpecTargetOrigin": "V",
  "ViolCount": 0,
  "ExtMaMV": 0.821,
  "ExtEwmaMV": 0.823928968184205,
  "ExtMsMV": 0.00494974746830584,
  "OutOfOrder": "N",
  "EwmaS": 0.0294076145612229,
  "EwmaR": 0.041588647349522,
  "Calc2ID": 4240949,
  "LimitEnable": "YYNYYNYYYYYYYYYYYYYYYYYYYYYYNNNNNNNNN",
  "ExtEwmaMVLCL": 0.707426333900759,
  "ExtEwmaMVCenter": 0.772760751059963,
  "ExtEwmaMVUCL": 0.838095168219167,
  "ExtMaMVLCL": 0.726562341641818,
  "ExtMaMVCenter": 0.772760751059963,
  "ExtMaMVUCL": 0.818959160478108,
  "EwmaSLCL": 0.0145134801150431,
  "EwmaSCenter": 0.0339539741967941,
  "EwmaSUCL": 0.0533944682785451,
  "EwmaRLCL": 0.0205251604159266,
  "EwmaRCenter": 0.0480181708055726,
  "EwmaRUCL": 0.0755111811952186,
  "ExtMSMVLCL": 4.82494288057976E-05,
  "ExtMSMVCenter": 0.0227529820203257,
  "ExtMSMVUCL": 0.0913996099236622,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 0.545228027822357,
  "RawUal": 1.00029347429757,
  "SigmaLal": 0.0,
  "SigmaUal": 0.0,
  "RangeLal": 0.0,
  "RangeUal": 0.0,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND 4.5 Sigma",
  "ChannelType": "N",
  "ChState": "Online"
}