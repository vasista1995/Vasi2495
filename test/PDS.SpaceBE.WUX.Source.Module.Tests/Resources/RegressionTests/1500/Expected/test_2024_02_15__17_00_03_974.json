{
  "IdSource": "SPACEACT2:WUX:BE:ZF323051C06:TCWUS:P_Test_Scan_Pack:Peel_Of_Force:50614:200697763:0",
  "Data_flat_Metadata": {
    "Package": "PG-TDSON-8-66",
    "ParameterClass": "E_SPC",
    "Segment": "TDSON8",
    "Module": "M2_TEST",
    "FourDReport": "*",
    "ChipType": "P9222C21311",
    "SalesName": "BSC093N15NS5",
    "Track": "T1",
    "Operator": "30040342",
    "Baunumber": "90043423",
    "Line": "TDSON8",
    "Shift": "C",
    "Process": "Testing",
    "Lot": "ZF323051C06",
    "ParameterName": "Peel_Of_Force",
    "ParameterUnit": "N",
    "ParameterOper": "P_Test_Scan_Pack",
    "ParameterFacility": "TCWUS",
    "SiteKey": "WUX",
    "ChannelId": "50614",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-06T07:16:52Z",
    "CreatedTimestamp": "2023-10-06T15:16:52Z",
    "UpdatedTimestamp": "2023-10-06T15:16:52Z",
    "UpdatedTimestampUtc": "2023-10-06T07:16:52Z",
    "ExportedTimestampUtc": "2023-10-06T15:16:52Z",
    "ExportedTimestamp": "2023-10-06T15:16:52Z",
    "SampleTimestampUtc": "2023-10-06T07:16:52Z",
    "SampleTimestamp": "2023-10-06T15:16:52Z",
    "SourceDataLevel": "C",
    "Equipment": "SRMS04",
    "ChannelName": "Tape_PeelOffForce/TDSON8",
    "ChannelDescr": "Created from LDS \nexclude list by \naction 'Create SPC Channel'",
    "RvStoreFlag": "Y",
    "CalcCreated": "11/22/2022 16:25:53",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 4240949,
    "ViolCount": 0,
    "CalculationStrategy": "ND 4.5 Sigma",
    "LdsID": "1500",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 1.1,
        "SpecLow": 0.4,
        "SpecTarget": 0.75,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYYNYYYYYYYYYYYYYYYYYYYYYYNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": ",8325",
        "Median": ",8325",
        "Sigma": ",0233345237791561",
        "Min": ",816",
        "Max": ",849",
        "Q1": ",82425",
        "Q3": ",84075"
      },
      "ControlLimits": {
        "ctrl_mean_low": 0.576785692454335,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 0.772760751059963,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 0.968735809665591,
        "CtrlMeanHighEnabled": "Y",
        "ctrl_low": 0.545228027822357,
        "CtrlLowEnabled": "Y",
        "CtrlSigmaCenterEnabled": "Y",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "Y",
        "ctrl_target": 0.772760751059963,
        "ctrl_high": 1.00029347429757,
        "CtrlHighEnabled": "Y",
        "CtrlSigmaLowEnabled": "N",
        "ctrl_sigma_target": 0.0363419972369961,
        "ctrl_sigma_high": 0.168815409461328,
        "CtrlSigmaHighEnabled": "Y",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.0480181708055726,
        "ctrl_range_high": 0.2796513136687,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 0.821,
        "ExtEwmaMV": 0.823928968184205,
        "ExtMsMV": 0.00494974746830584,
        "EwmaS": 0.0294076145612229,
        "EwmaR": 0.041588647349522,
        "ExtEwmaMVLCL": 0.707426333900759,
        "ExtEwmaMVCenter": 0.772760751059963,
        "ExtEwmaMVUCL": 0.838095168219167,
        "ExtMaMVLCL": 0.726562341641818,
        "ExtMaMVCenter": 0.772760751059963,
        "ExtMaMVUCL": 0.818959160478108,
        "EwmaSLCL": 0.0145134801150431,
        "EwmaSCenter": 0.0339539741967941,
        "EwmaSUCL": 0.0533944682785451,
        "EwmaRLCL": 0.0205251604159266,
        "EwmaRCenter": 0.0480181708055726,
        "EwmaRUCL": 0.0755111811952186,
        "ExtMSMVLCL": 4.82494288057976E-05,
        "ExtMSMVCenter": 0.0227529820203257,
        "ExtMSMVUCL": 0.0913996099236622,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.545228027822357,
        "RawUal": 1.00029347429757,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 200697763,
        "Seqnr": 2,
        "Value": 0.849,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 200697763,
        "Seqnr": 1,
        "Value": 0.816,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 2
  },
  "ProductionAction": {
    "Id": "SPACEACT1:WUX:BE:TCWUS:P_Test_Scan_Pack:Peel_Of_Force:50614",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:ZF323051C06",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "WUX:BE:ZF323051C06:TCWUS:P_Test_Scan_Pack:Peel_Of_Force:50614:200697763:0"
  }
}